cmake/modules/FindLibcloudproviders.cmake

```diff
@@ -17,6 +19,15 @@ find_library(LIBCLOUDPROVIDERS_LIBRARY
        ${CMAKE_INSTALL_PREFIX}/lib
 )
 
-message("================> ${LIBCLOUDPROVIDERS_LIBRARY}")
+find_package_handle_standard_args(Libcloudproviders
+  DEFAULT_MSG
+  Libcloudproviders_INCLUDE_DIR
+  Libcloudproviders_LIBRARY
+  )
+
+if (Libcloudproviders_FOUND)
+  set(Libcloudproviders_INCLUDE_DIRS ${Libcloudproviders_INCLUDE_DIR})
```

Comment:
Since we're at it, it would be even better to declare proper targets, for instance this way we wouldn't even need to mess with include_directories anymore for that dependency.